server.js

```diff
@@ -4739,6 +4739,53 @@ cache(function(data, match, sendBadge, request) {
   }
 }));
 
+// Issue Stats integration.
+camp.route(/^\/issuestats\/([^\/]+)\/(.+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Could you add a 1-letter indicator for this kind of badge? We usually have one, to allow for future badges coming from the same vendor. For instance, `/npm/v/…` for versions, `/l/` for licence…